application/database/db.py

```diff
@@ -325,70 +366,94 @@ def gap_analysis(self, name_1, name_2):
-                    "sectionID": seg.end_node["section_id"],
-                    "section": seg.end_node["section"],
-                    "subsection": seg.end_node["subsection"],
-                    "description": seg.end_node["description"],
-                    "id": seg.end_node["id"],
-                },
+                "start": self.parse_node(seg.start_node),
+                "end": self.parse_node(seg.end_node),
                 "relationship": seg.type,
             }
 
         def format_path_record(rec):
             return {
-                "start": {
-                    "name": rec.start_node["name"],
-                    "sectionID": rec.start_node["section_id"],
-                    "section": rec.start_node["section"],
-                    "subsection": rec.start_node["subsection"],
-                    "description": rec.start_node["description"],
-                    "id": rec.start_node["id"],
-                },
-                "end": {
-                    "name": rec.end_node["name"],
-                    "sectionID": rec.end_node["section_id"],
-                    "section": rec.end_node["section"],
-                    "subsection": rec.end_node["subsection"],
-                    "description": rec.end_node["description"],
-                    "id": rec.end_node["id"],
-                },
+                "start": self.parse_node(rec.start_node),
+                "end": self.parse_node(rec.end_node),
                 "path": [format_segment(seg) for seg in rec.relationships],
             }
 
-        def format_record(rec):
-            return {
-                "name": rec["name"],
-                "sectionID": rec["section_id"],
-                "section": rec["section"],
-                "subsection": rec["subsection"],
-                "description": rec["description"],
-                "id": rec["id"],
-            }
-
-        return [format_record(rec["BaseStandard"]) for rec in base_standard], [
+        return [self.parse_node(rec["BaseStandard"]) for rec in base_standard], [
             format_path_record(rec["p"]) for rec in (path_records + path_records_all)
         ]
 
     @classmethod
-    def standards(self):
+    def standards(self) -> List[str]:
         if not self.connected:
             return
         records, _, _ = self.driver.execute_query(
-            'MATCH (n:Node {ntype: "Standard"}) ' "RETURN collect(distinct n.name)",
+            "MATCH (n:Standard|Tool) " "RETURN collect(distinct n.name)",
             database_="neo4j",
         )
         return records[0][0]
 
+    @classmethod
+    def parse_node(self, node: neo4j.graph.Node) -> cre_defs.Document:
+        name = node["name"]
+        id = node["id"] if "id" in node else None
+        description = node["description"] if "description" in node else None
+        # links = [self.parse_link(link) for link in node["links"]]
+        tags = node["tags"]
+        # metadata = node["metadata"]
+        if "Code" in node.labels:
```

Comment:
please do not hardcode the node types here, instead it should be `if cre_defs.doctypes.Code.value in node.labels:` this way we can change the doctypes easily without having to match and replace